src/main/java/com/rarchives/ripme/ripper/rippers/HentaiNexusRipper.java

```diff
@@ -120,6 +121,7 @@ The following code is a Java adaptation of the initRender() JavaScript function
 
         byte[] jsonBytes = Base64.getDecoder().decode(jsonEncodedString);
 
+        @SuppressWarnings("rawtypes")
         ArrayList unknownArray = new ArrayList();
```

Comment:
Don't suppress warnings if possible. Figure out what was intended and update the code accordingly.

Is it possible to update this type so that it's a properly templated type?

In this case, it looks to me like ArrayList<Integer> is the appropriate type.

Reply:
Again, please don't suppress these warnings. Fix them.